### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by ChGen**
Updated docs. Added current performance test rig documentation. Part of #64

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus @olenagerasimova , please review.

File: loadtests/perftests-setup.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+#### Graphs for test tags
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-files-ul.svg
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-files-dl.svg
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-files-maven-ul.svg
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-files-maven-dl.svg
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-maven-ul.svg
+- https://central.artipie.com/artipie/benchmarks/perftests_repo/graphs_t/jmx-maven-dl.svg
+
+#### Graphs for test tags
```
Comment (olenagerasimova) [inline loadtests/perftests-setup.md]: this header seems off

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@ChGen thanks, one comment

File: loadtests/perftests-setup.md (hunk 1 above)
Comment (ChGen) [reply loadtests/perftests-setup.md]: Thanks

**Review by olenagerasimova — APPROVED** [review]
lgtm

**Review by dgarus — APPROVED** [review]
lgtm

**Pull request merged**